code/game/gamemodes/endgame/bluespace_jump/bluespace_jump.dm

```diff
@@ -8,7 +8,7 @@
 	affected_levels = zlevels
 
 /datum/universal_state/bluespace_jump/OnEnter()
-	var/space_zlevel = current_map.get_empty_zlevel() //get a place for stragglers
+	var/space_zlevel = current_map.get_empty_zlevel("blue", "space") //get a place for stragglers
```

Comment:
This isn't gonna work. It needs to be a pregenerated zlevel otherwise it's gonna lagspike during transfer since it'll have to make a new zlevel.

Reply:
added to SSatlas generation